presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1024,6 +1040,17 @@ private String validateAndNormalizeAvroSchemaUrl(String url, HdfsContext context
         }
     }
 
+    private String validateAvroSchemaLiteral(String avroSchemaLiteral)
+    {
+        try {
+            new Schema.Parser().parse(avroSchemaLiteral);
+            return avroSchemaLiteral;
+        }
+        catch (SchemaParseException e) {
```

Comment:
Can there be any other exception flying here?

Reply:
By looking at the implementation of `Schema.Parser::parse()`, I think `SchemaParseException` is sufficient here